### PR TITLE
Update to bdk v0.26.0

### PR DESCRIPTION
**Pull request by rajarshimaitra**
### Description

Upgrade to bdk v0.26.0 

#### All Submissions:

* [x] I've signed all my commits
* [x] I followed the [contribution guidelines](https://github.com/bitcoindevkit/bdk-cli/blob/master/CONTRIBUTING.md)
* [x] I ran `cargo fmt` and `cargo clippy` before committing

### CODE REVIEW

**Comment by notmandatory** [comment]
Hey looks like you need do to a `cargo fmt`, missing a comma.

**Comment by rajarshimaitra** [comment]
Ah my bad.. Updated..

**Comment by notmandatory** [comment]
I updated bdk-reserves "update_bdk_026" branch with bdk rc.2.

File: src/utils.rs [hunk 1]
```diff
@@ -390,6 +390,7 @@ pub(crate) fn new_blockchain(
             retry: wallet_opts.proxy_opts.retries,
             timeout: wallet_opts.electrum_opts.timeout,
             stop_gap: wallet_opts.electrum_opts.stop_gap,
+            validate_domain: false,
```
Comment (notmandatory) [inline src/utils.rs:393]: This should be true since that's the default prior to adding the option.
```suggestion
            validate_domain: true,
```

Comment (rajarshimaitra) [reply src/utils.rs]: ACK.. Missed that thought..

**Review by notmandatory — APPROVED** [review]
ACK 52dd87efa78d312092faa625fd5756a1dc9c596f

**Pull request merged**